README.md

```diff
@@ -84,18 +84,18 @@ or cloned via
 ```
 
 
- * Brief instructions to set up a build system on Windows can be found [in the Crispy Doom Wiki](./wiki/build-windows). A much more detailed guide is provided [in the Chocolate Doom Wiki](https://www.chocolate-doom.org/wiki/index.php/Building_Chocolate_Doom_on_Windows), but applies to Crispy Doom as well for most parts.
- * There are also instructions for building on [Linux](./wiki/build-linux) and [MacOS](./wiki/build-macos)
+ * Brief instructions to set up a build system on Windows can be found [in the Crispy Doom Wiki](https://github.com/fabiangreffrath/crispy-doom/wiki/Building-on-Windows). A much more detailed guide is provided [in the Chocolate Doom Wiki](https://www.chocolate-doom.org/wiki/index.php/Building_Chocolate_Doom_on_Windows), but applies to Crispy Doom as well for most parts.
+ * There are also instructions for building on [Linux](https://github.com/fabiangreffrath/crispy-doom/wiki/Building-on-Linux) and [MacOS](https://github.com/fabiangreffrath/crispy-doom/wiki/Building-on-Mac)
 
 
 ## Documentation
 
- * **[New Cheat Codes](./wiki/cheats)**
- * **[New Command-Line Parameters](./wiki/params)**
- * **[New Controls](./wiki/controls) (With default bindings)**
- * **[Crispness](./wiki/crispness)**
- * **[Compatibility](./wiki/compatibility)**
- * **[FAQ](./wiki/faq)**
+ * **[New Cheat Codes](https://github.com/fabiangreffrath/crispy-doom/wiki/New-Cheats)**
+ * **[New Command-Line Parameters](https://github.com/fabiangreffrath/crispy-doom/wiki/New-Command-line-Parameters./wiki/params)**
```

Comment:
trailing ./wiki/params

Reply:
Thanks, fixed in https://github.com/fabiangreffrath/crispy-doom/pull/1054/commits/47bb9602ed2a9a4b8a9749065ee056888eb7a12f